.gitignore

```diff
@@ -107,7 +107,7 @@ detox/detox_pixel_4_xl_api_30
 #editor-settings
 .vscode
 .scannerwork
+launch.json
```

Comment:
why is it here ?

Reply:
I accidentally committed my vscode launch.json config; have it setup to debug from the IDE.